ovs-p4rt/ovs_p4rt.cc

```diff
@@ -1642,6 +1626,8 @@ void ConfigTunnelSrcPortTableEntry(struct src_port_info tnl_sp,
   status = ovs_p4rt::SendWriteRequest(session.get(), write_request);
 
   if (!status.ok()) return;
+
+  return;
```

Comment:
This statement is a no-op. Can be removed

Reply:
This:

```c
status = ovs_p4rt::SendWriteRequest(session.get(), write_request);
if (!status.ok()) return;
```

doesn't make much sense either. At the very least, there should be a comment explaining why the code is discarding the information that the `SendWriteRequest` failed.